mailserver.env

```diff
@@ -400,12 +400,6 @@ SA_TAG2=6.31
 # triggers spam evasive actions
 SA_KILL=10.0
 
-# add tag to subject if spam detected
-# The value `undef` opts-out of this feature. The value shown below is the default.
-# NOTE: By default spam is delivered to a junk folder, reducing the value of adding a subject prefix.
-# NOTE: If not using Docker Compose, other CRI may require the single quotes removed.
-#SA_SPAM_SUBJECT='***SPAM*** '
-
```

Comment:
This file doesn't have a `SPAM_SUBJECT` replacement?

The web UI won't allow me to provide a suggestions here, but you'd want to commit this:

```env
# Add the value as prefix to the mail subject when spam is detected
# NOTE: By default spam is delivered to a junk folder, reducing the value of a subject prefix for spam.
# NOTE: If not using Docker Compose, other CRI may require the single quotes removed.
SPAM_SUBJECT=
```

Reply:
I had #3791 in mind, but I guess doing it with 1 PR is the best course of action.